cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Insert.hs

```diff
@@ -69,6 +71,11 @@ import qualified Shelley.Spec.Ledger.PParams as Shelley
 import qualified Shelley.Spec.Ledger.TxBody as Shelley
 
 
+-- TODO(KS): We should probably move this to someplace better.
```

Comment:
There should not be two copies of this code.

Reply:
Fixed.